lib/beaker/ssh_connection.rb

```diff
@@ -267,6 +267,7 @@ def scp_to source, target, options = {}
         logger.warn "#{e.class} error in scp'ing. Forcing the connection to close, which should " <<
           "raise an error."
         close
+        raise e
```

Comment:
The `warn` message here seems to indicate that the forced closure of the SSH connection should raise an error; is that getting swallowed up somewhere and not raising?

Reply:
As far as I can tell, the `close` methods don't actually raise failures (unless the close itself fails):
https://github.com/puppetlabs/beaker/blob/master/lib/beaker/ssh_connection.rb#L95 which calls https://github.com/puppetlabs/beaker/blob/master/lib/beaker/host.rb#L313 I believe.

It seems like the `close` methods *shouldn't* raise an error - like what if you want to close just because you're done and not because something went wrong?